website/www/site/content/en/documentation/transforms/python/aggregation/groupbykey.md

```diff
@@ -16,25 +16,40 @@ limitations under the License.
 -->
 
 # GroupByKey
-<table align="left">
-    <a target="_blank" class="button"
-        href="https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.core.html#apache_beam.transforms.core.GroupByKey">
-      <img src="https://beam.apache.org/images/logos/sdks/python.png" width="20px" height="20px"
-           alt="Pydoc" />
-     Pydoc
-    </a>
-</table>
-<br><br>
 
+{{< localstorage language language-py >}}
+
+{{< button-pydoc path="apache_beam.transforms.core" class="GroupByKey" >}}
 
 Takes a keyed collection of elements and produces a collection
 where each element consists of a key and all values associated with that key.
 
 See more information in the [Beam Programming Guide](/documentation/programming-guide/#groupbykey).
 
 ## Examples
-See [BEAM-7390](https://issues.apache.org/jira/browse/BEAM-7390) for updates. 
 
-## Related transforms 
+In the following example, we create a pipeline with a `PCollection` of produce keyed by season.
+
+We use `GroupByKey` group all the produce for each season.
```

Comment:
Missing a "to"

We use `GroupByKey` *to* group all the produce for each season.

Reply:
Thanks, done